openapi/components/schemas/Subscription/OrderTypes/subscription-order.yaml

```
@@ -37,6 +37,10 @@ allOf:
             description: The time the trial should end
             type: string
             format: date-time
+      isTrialOnly:
+        description: Whether a subscription churns after a trial period.
```

```suggestion
        description: Whether a subscription ends after a trial period.
```

---

```suggestion
        description: Whether a subscription ends after a trial period. Recurring settings are ignored if it's `true`.
```
No precision is excessive :slightly_smiling_face: